docs/src/pages/components/data-grid/rows/rows.md

```diff
@@ -15,6 +15,12 @@ The rows can be defined with the `rows` prop, which expects an array of objects.
 
 {{"demo": "pages/components/data-grid/rows/RowsGrid.js", "bg": "inline"}}
 
+Note that `DataGrid` requires all rows to have a unique `id` property.
```

Comment:
> Note

I think we should either use `**Note**:` or `> ⚠️` (depending on how much we want to emphases this warning).

> you can pass `getRowId` prop to generate

I feel like `generate` encourages people to literally generate a random id, which is not a good idea if they want to do any type of update afterwards. 

```suggestion
> ⚠️ Each row should have an uniq id.
> By default, the grid will use the `id` property of the row.
> 
> When using dataset without an uniq `id` property, you can use the `getRowId` prop to specify a custom id for each row.
> ```tsx
>   <DataGrid getRowId={(row) => row.internalId} />
> ```
```

Reply:
Good point!
I've removed demo preview, since I believe inline example is enough, but I'm keeping the demo so that users can refer to a working example if needed.